src/main/java/kitchenpos/menus/tobe/application/MenuService.kt

```diff
@@ -0,0 +1,82 @@
+package kitchenpos.menus.tobe.application
+
+import kitchenpos.common.Price
+import kitchenpos.menus.tobe.domain.*
+import kitchenpos.menus.tobe.dto.`in`.MenuCreateRequest
+import kitchenpos.menus.tobe.dto.out.MenuResponse
+import kitchenpos.menus.tobe.dto.out.fromMenu
+import kitchenpos.products.domain.ProductPriceChanged
+import kitchenpos.products.tobe.domain.ProductRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Propagation
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.transaction.event.TransactionPhase
+import org.springframework.transaction.event.TransactionalEventListener
+import java.util.*
+
+@Service
+class MenuService(
+    private val menuRepository: MenuRepository,
+    private val menuGroupRepository: MenuGroupRepository,
+    private val menuNameValidatorService: MenuNameValidatorService,
+    private val menuPriceValidatorService: MenuPriceValidatorService,
+    private val productRepository: ProductRepository,
+) {
+
+    @Transactional
+    fun createMenu(request: MenuCreateRequest): MenuResponse {
+        val menuGroup = menuGroupRepository.findByIdOrNull(request.menuGroupId)
+            ?: throw NoSuchElementException("can not found menu group")
```

Comment:
사소할 수 있지만 어떤 값으로 조회했는지를 예외 메시지에 포함한다면 에러를 파악하기가 조금 더 쉬울 것 같아요. 🙂
```suggestion
            ?: throw NoSuchElementException("can not found menu group.  menuGroupId: ${request.menuGroupId}")
```

Reply:
좋은 의견 감사합니다 ㅎㅎ